tasks.py

```diff
@@ -210,3 +209,30 @@ def run(context, command):
             {command}
         """,
     )
+
+@task
+def bitfield(context,input, output, lanes, bits):
```

Comment:
Please introduce these tasks in a separate `templates/DO-178C/tasks.py` file so that we don't mix the general tasks of this repo and the DO-178C-specific tasks.

In my comment above, I am asking this question which also connects to this topic:

> One way to overcome this would be to create a completely standalone repository dedicated to DO-178C. Then we would not have any conflicts with other standards. Thinking about this now, I kind of like this idea. What do you think?